src/routes/Agm.jsx

```diff
@@ -0,0 +1,106 @@
+import { Close, CopyAll, LinkOutlined, OpenInNew } from "@mui/icons-material";
+import { Box, Button, Dialog, IconButton, Stack } from "@mui/material";
+import React from "react";
+import "../App.css";
+import "../assets/agm.css";
+
+function Agm() {
+  const [isDialogOpen, setIsDialogOpen] = React.useState(false);
+  return (
+    <div className="content-wrapper">
+      <div className="edges" style={{ color: "#111", fontFamily: "Montserrat" }}>
+        <div style={{ maxWidth: "80ch" }}>
+          <h1 style={{ padding: 0, margin: "0 0 0.7rem 0" }}>Annual General Meeting (AGM) 2024</h1>
+          <p style={{ padding: 0, margin: 0, fontSize: "1.4rem" }}>
+            Have your say on our 2025 executive team and nominate yourself or others to be in the 2025 Code Network Executive team.
```

Comment:
This seems a bit repetitive. How about "Have your say on our club's direction and..."